### ISSUE TITLE
Which Clover/MacOS version was used?

### ISSUE THREAD

**Issue by BRichter99** [fredldotme/Dell-XPS13-9343-OpenCore#3]
Hi, I'm trying to make MacOS work on my i7 XPS9343. The goal is Monterey, but first I want to make an initial working installation.

Do you remember which version of MacOS and Clover you had installed?

**Comment by Jojoello** [comment]
Of course, thank you very much for your work. I installed MacOS Big Sur
11.6.6 20G624.And I already update to MacOS Big Sur 11.7.1，Its perfect !

BRichter99 ***@***.***> 于2022年11月23日周三 06:11写道：

**Comment by Jojoello** [comment]
https://github.com/frec0x/xps13-9343-clover-bigsur     Try this

Joel Lo ***@***.***> 于2022年11月25日周五 16:23写道：

**Comment by BRichter99** [comment]
Thank you very much for the suggestion. However, I cannot boot my Big Sur Installer with your EFI. Is your computer also the one with an i7 CPU?

 I get stuck at the apple logo with about 1/3 of the progress bar. With boot option -v it shows
_apfs_module_start:2487: load: com.apple.filesystem.apfs, v1677.141.3, apfs-1677.141.3.7.2, 2022/06/28_
as the last message. I haven't figured out how to generate a log file to provide here. Do you have any suggestions?

**Comment by Jojoello** [comment]
Try to update the clover to the newest version, and wait longer during
installation,this EFI is easy to install

BRichter99 ***@***.***>于2022年11月29日 周二07:08写道：

**Comment by Jojoello** [comment]
I gotta xps13 9343 with i7 and 5500hdGPU,8with 128gb


BRichter99 ***@***.***>于2022年11月29日 周二07:08写道：
